tests/model/test_model.py

```diff
@@ -1870,11 +1871,15 @@ def test_update_star_status_invalid_operation(
             ("add", 1, ["read"], ["read", "starred"]),
             ("add", 1, ["starred"], ["starred"]),
             ("add", 1, ["read", "starred"], ["read", "starred"]),
+            ("add", 1, ["mentioned"], ["mentioned", "starred"]),
+            ("add", 1, ["wildcard_mentioned"], ["wildcard_mentioned", "starred"]),
             ("remove", -1, [], []),
             ("remove", -1, ["read"], ["read"]),
             ("remove", -1, ["starred"], []),
             ("remove", -1, ["read", "starred"], ["read"]),
             ("remove", -1, ["starred", "read"], ["read"]),
+            ("remove", -1, ["mentioned", "starred"], ["mentioned"]),
+            ("remove", -1, ["wildcard_mentioned", "starred"], ["wildcard_mentioned"]),
```

Comment:
These extra cases in this commit are good examples, but are giving quite random coverage that I'm not sure helps dramatically. Think of what the other cases cover.

Reply:
I'm not sure if the current iteration of my PR addresses this review, but I tried to cover a few other cases that the previous tests were missing. Let me know if these new cases are still not what you were expecting. :)